.github/workflows/tests-e2e-upgrade.yaml

```diff
@@ -158,6 +160,8 @@ jobs:
             lb-mode: 'snat'
             egress-gateway: 'true'
             lb-acceleration: 'testing-only'
+            # Disable until https://github.com/cilium/cilium/issues/30717
+            # has been resolved.
             ingress-controller: 'true'
```

Comment:
This doesn't disable ingress-controller. :thinking:

Reply:
There's several more cases like this. cc @brlbil between author and reviewer, we should be catching this stuff. At least for me, this becomes immediately apparent when reviewing commit-by-commit.

Reply:
@joestringer you are right. It seems that the midnight review is not a good idea after all.

Reply:
:sweat_smile: Live and learn. Looks like @brb is already on top of it: https://github.com/cilium/cilium/pull/31917